Enigmatry.Entry.AspNetCore.Tests/SampleAppTests/SampleAppFixtureBase.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Enigmatry.Entry.AspNetCore.SampleApp;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Enigmatry.Entry.AspNetCore.Tests.SampleAppTests;
+
+[System.Diagnostics.CodeAnalysis.SuppressMessage("Design",
+    "CA1001:Types that own disposable fields should be disposable",
+    Justification = "Fields are disposed in Teardown method")]
+public abstract class SampleAppFixtureBase
+{
```

Comment:
Personally, I find inheritance (base testing class) very inconvenient technique for writing tests. Tests are inflexible, hard to read and after you get decent number of them in your suite, you need to know too much about base test class than you need to. I'm not the only one who thinks so (i.e):
https://www.petrikainulainen.net/programming/unit-testing/3-reasons-why-we-should-not-use-inheritance-in-our-tests/
https://www.testingdocs.com/disadvantages-of-using-inherited-tests-in-junit/

Oh, Zoran Horvat also thinks this approach is obsolete :)

Reply:
In this case inheritance served the purpose well. You are open to refactor tests as you like it. The important thing here is that we can now easily write tests to see how our code integrates with the running application without having to fake and mock asp net core internals.

Reply:
I did already some job for composition approach in spike for integration testing. Once I create PR for that, I'll change this code as well. Just wanted to raise awareness.